### ISSUE TITLE
meta-DAO options

### ISSUE THREAD

**Issue by owocki** [gitcoinco/ERC-1789#25]
It may be nice to try a few different daos at the same time as a trial. Splitting inflation revenue

* moloch style
* aragon
* DAOstack
* colony
* simple multisig wallet
* CLR matching

what else?

**Comment by lrettig** [comment]
Right, so one way to think about _this_ initiative is as a "meta-DAO" that's responsible for a.) coordination/liaising with core devs and teams seeking funding, b.) responsible stewardship and allocation of funding across multiple lower-level DAOs. Sort of like a "fund of funds" approach. Maybe what we need to propose here is a "faucet" contract that receives the block reward and allocates it proportionally to set of lower-level contracts such as these, with governance over it.

**Comment by lrettig** [comment]
To be clear one advantage of doing things this way is that it achieves one of my goals for this project which is to minimize custody over actual funds. I'm still not convinced this is achievable in the first few phases but it's worth aiming for nonetheless.

**Comment by MadeofTin** [comment]
> Maybe what we need to propose here is a "faucet" contract that receives the block reward and allocates it proportionally to set of lower-level contracts such as these, with governance over it.

Very Very Plus one to this!

A meta-DAO that decides a few DAOs/Orgs to get a proportion of the funds.

**Comment by MadeofTin** [comment]
> It may be nice to try a few different daos at the same time as a trial. Splitting inflation revenue
> 
> * moloch style
> * aragon
> * DAOstack
> * colony
> * simple multisig wallet
> * CLR matching
> 
> what else?

DAOstack Dev dao governed by previous recipients of the grants.

I would love for a portion of the funds to be as "in the devs hands" as possible.

Rick Dudleys EthCC talk is a big inspiration on this idea for me. The link goes to the relevant section.

https://youtu.be/GOkSg0BuSdw?t=1270

**Comment by kronosapiens** [comment]
You could even avoid needing the "dao of daos" by hard-coding the distribution of inflation going to each of the daos, i.e. at each "hard fork" in which you reset the inflation rate (IIRC that was the idea), you also include the set of recipient addresses and the % of the inflation they receive. Once the inflation decays away, we assess performance and decide which daos to include in the next hard fork.

**Comment by lrettig** [comment]
> You could even avoid needing the "dao of daos" by hard-coding the distribution of inflation going to each of the daos, i.e. at each "hard fork"

This idea troubles me a little bit since the hard fork is our nuclear governance mechanism and it's relatively slow, expensive, and rare, happening on the order of 1-2x per year, whereas I could see wanting or needing to tweak relative per-DAO allocations more frequently. I could see a small amount of funding being allocated to a given sub-DAO, and that relative amount increasing as it demonstrates progress and builds trust. I could also see a scenario where we might need to cut off a particular DAO if something goes wrong, and it would be very nice to be able to do this without needing a network-wide hard fork.

In general I think we should push as much of this to layer two as possible.

**Comment by lrettig** [comment]
I also just learned that Gnosis did a lot of work on this idea, which they called DaoZero. I'm in touch with them and will study and share their learnings and materials.

**Comment by kronosapiens** [comment]
I hear you, but my critique of that approach is that it makes the "meta-dao" a single point of failure. I'd argue that the risk of some of the daos misspend their funds is lesser than the risk of having a dysfunctional meta-dao undermine the entire experiment. I'd say 6 months is perhaps the minimum amount of time I'd want this to run anyway, so I don't think the hard fork cadence is inappropriate for this. I'd say that by giving all the daos the guarantee that they have at least that much time to work will allow them to experiment and learn from their mistakes, versus being perpetually at risk of having funding withdrawn for political reasons.

To put it another way, if the "risk" of any individual dao failing is between 1 and 10, then having two layers of daos means the "risk" is now between 1 and 100 since the failures cascade. I.e. if we have 7 daos, then the risk of failing with one layer is the sum of the individual risk, so between 0 and 70. If we have two layers, then the risk is that first risk times the risk of the meta-dao, so between 0 and 700, which is way worse. Although I just made up that model so I wouldn't take it too seriously.

**Comment by kronosapiens** [comment]
Also, separately, I like the idea of having a "burn funds" option be available to all the daos, if they feel like nothing is in need of funding. I like this because it avoids the problem of "spending money just because you have it"; giving a dao the option to burn some of the funds makes all of the other disbursements more meaningful and sends a good message (i.e. we're not just giving money away for nothing, your work has to be more valuable than the downside of inflating the eth supply).

**Comment by lrettig** [comment]
> I'd say 6 months is perhaps the minimum amount of time I'd want this to run anyway, so I don't think the hard fork cadence is inappropriate for this

Actually I think six months is way, way too long. We need the ability to throttle a DAO on a roughly weekly basis if we're not seeing progress, especially early on. The more I think about it, the more I think the DAICO model could be good for this.

I do like the "burn funds" idea, but I don't realistically see anyone using it, unless we can somehow enforce it.

**Comment by kronosapiens** [comment]
> We need the ability to throttle a DAO on a roughly weekly basis if we're not seeing progress, especially early on.

I'm deeply skeptical of this, I think that having such short timelines will keep this experiment from generating any useful learnings, but open to hearing what others think. Imagine if new engineering hires were fired after a week for not shipping any code. And folks don't have to use the "burn funds" option, but I think it's valuable just by being there (otherwise each DAO will be compelled to give funds away).

**Comment by jpitts** [comment]
I think that splitting the funds up across different DAOs is a great idea. Let experiments happen and we can all learn what works best. But the key is really to encourage diversity and some specialization in "fund allocator" DAOs.

**Comment by jpitts** [comment]
@owocki, I would also add "CREDO" to that list. It is a DAO / economic system which leverages DAI but features an equity token. CREDO stands for "Credit Reserve Economics for Decentralized Organizations" . 

@pet3r-pan is also becoming familiar with it.

I expect their white paper to come out shortly. In the meantime, this is a presentation on CREDO: https://www.youtube.com/watch?v=l17NjaRV4yk
